src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
+  const dateObj = {};
+
+  for (let i = 0; i < 3; i++) {
+    dateObj[fromFormat[i]] = dateParts[i];
+  }
+
+  if (fromFormat.includes('YYYY') && toFormat.includes('YY')) {
+    dateObj['YY'] = dateObj['YYYY'].slice(-2);
+  }
+
+  if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
+    if (dateObj.YY < 30) {
+      dateObj['YYYY'] = '20' + dateObj['YY'];
+    } else {
+      dateObj['YYYY'] = '19' + dateObj['YY'];
+    }
```

Comment:
Consider using a ternary operator for this conditional assignment to make the code more concise. For example: `dateObj['YYYY'] = dateObj['YY'] < 30 ? '20' + dateObj['YY'] : '19' + dateObj['YY'];`

Reply:
It makes sense - ternary you could use ternary here
FYI